config/config-mysql_apache_jvm_statsd.yaml

```diff
@@ -128,45 +146,56 @@ processors:
             new_label: usage_kind
             new_value: committed
 
-      - metric_name: java_lang_Memory_HeapMemoryUsage_used
+      - metric_name: java_lang_Memory_NonHeapMemoryUsage_max
         action: update
-        new_name: agent.googleapis.com/jvm/memory/usage
+        new_name: jvm/memory/usage
         operations: 
           - action: add_label
             new_label: memory_type
-            new_value: heap
+            new_value: non_heap
           - action: add_label
             new_label: usage_kind
-            new_value: used
+            new_value: max
 
-      - metric_name: java_lang_Memory_NonHeapMemoryUsage_max
+      - metric_name: java_lang_Memory_HeapMemoryUsage_used
         action: update
-        new_name: agent.googleapis.com/jvm/memory/usage
+        new_name: jvm/memory/usage
         operations: 
           - action: add_label
             new_label: memory_type
-            new_value: non_heap
+            new_value: heap
           - action: add_label
             new_label: usage_kind
-            new_value: max
+            new_value: used
 
       - metric_name: java_lang_Threading_DaemonThreadCount
         action: update
-        new_name: agent.googleapis.com/jvm/thread/num_daemon
+        new_name: jvm/thread/num_daemon
       
       - metric_name: java_lang_Threading_ThreadCount
         action: update
-        new_name: agent.googleapis.com/jvm/thread/num_live
+        new_name: jvm/thread/num_live
       
       - metric_name: java_lang_OperatingSystem_OpenFileDescriptorCount
         action: update
-        new_name: agent.googleapis.com/jvm/os/open_files
+        new_name: jvm/os/open_files
```

Comment:
The above three metrics seem to be not be included in the filter but have transforms listed here, mistake?

Reply:
Also would be good to make the order of the metrics listed in the filter match the order in the transforms just so it's a bit easier to read

Reply:
nice catch! the rest should already be in order though!